scripts/release/run_binarydeb_job.py

```diff
@@ -69,6 +71,7 @@ def main(argv=sys.argv[1:]):
 
         'binarydeb_dir': '/tmp/binarydeb',
         'dockerfile_dir': '/tmp/docker_build_binarydeb',
+        'build_environment_variables': args.env_vars
```

Comment:
Nitpick: move this line one up so that it is passed in the order it is being used in the template.